inter-scheme-proxy-adapter/templates/deployment.yaml

```diff
@@ -21,6 +21,7 @@ spec:
   template:
     metadata:
       annotations:
+        reloader.stakater.com/auto: 'true'
```

Comment:
Is this k8s feature or depends on a new service?
If its not default k8s behaviour and this config is for some other dependent service, can't we use podAnnotations in the deployments instead of having technology specific configuration here?

Reply:
Ok, but its not part of mojaloop helm deployment. Isn't it? Why do we need it here by default instead of using podAnnotations?